src/pages/ScholarshipForm.jsx

```diff
@@ -38,9 +41,21 @@ function ScholarshipForm() {
         {requiredTextField('name', 'Scholarship Name')}
         {requiredTextField('description', 'Description')}
         {requiredTextField('website', 'Website')}
-        <Button variant="contained" color="primary">
-          Submit
-        </Button>
+        <EnumSelectField
+          id="scholarshipType"
+          label="Scholarship Type"
+          value={formFieldStates.scholarshipType}
+          onChange={updateFn('scholarshipType')}
+          enums={ScholarshipType}
+        />
+        <div>
+          <Button
+            variant="contained"
+            color="primary"
+            onClick={() => console.log(formFieldStates)}>
```

Comment:
Hmm, I think you can write the actual submission logic using `Scholarship.new()`. Though since we haven't figured out roles yet maybe we can add the `verificationStatus` enum + field and have it set it to Unverified.

Reply:
Or if you want to leave the functionality for a different PR/issue, maybe a [Snackbar](https://material-ui.com/components/snackbars/#customized-snackbars) might be a better way to indicate work is still in progress.

Reply:
I'm going to leave it for another PR, so that I can have all the required fields finished first. For now I'll put the SnackBar.